src/config/graphical-session/wayland.md

```diff
@@ -32,18 +32,22 @@ Void Linux currently packages the following Wayland compositors:
 Both GNOME and KDE Plasma have EGLStreams backends for Wayland, which means they
 can use the proprietary NVIDIA drivers. Most other Wayland compositors require
 drivers that implement the GBM interface. The main driver for this purpose is
-provided by the `mesa-dri` package. The [graphics
+provided by the `mesa-dri` package. The [graphics drivers
 section](./graphics-drivers/index.md) has more details regarding setting up
 graphics in different systems.
 
 ### Native applications
 
-Qt5 based applications require the installation of the `qt5-wayland` package and
-setting the environment variable `QT_QPA_PLATFORM=wayland-egl` to enable their
-Wayland backend. Some KDE specific applications might also require the
-installation of the `kwayland` package. GTK+ applications should do it
-automatically, but setting `GDK_BACKEND=wayland` can force the Wayland backend
-(however, this can break applications such as Chromium).
+[Qt5](https://wayland.freedesktop.org/qt5.html) based applications require the
+installation of the `qt5-wayland` package and setting the environment variable
+`QT_QPA_PLATFORM=wayland-egl` to enable their Wayland backend. Some KDE specific
+applications might also require the installation of the `kwayland` package.
+[EFL](https://wayland.freedesktop.org/efl.html) based applications require
+setting the environment variable `ELM_DISPLAY=wl`, and can have issues without
```

Comment:
Is ", and can have issues ..." required?

Reply:
I can be more specific about the issue, but I think it's necessary to mention it. Basically, EFL has some weird issue with XWayland, which it tries to use by default, and simply doesn't launch the application. It is necessary to set that environment variable for it to work at all (or unset `DISPLAY`, but that's a worse solution).

Reply:
No, my concern is not about going into the details of the specific issues, but that you've already said setting the env var is required, so is there any need to mention that it can have issues without it? Obviously we wouldn't be saying that it's required otherwise. :-)

Reply:
I wanted to avoid confusion regarding the word required: "required" to force the application to use Wayland, or "required" to make the application work at all. If you think it was clear already without all the explanation, I can certainly remove it.

Reply:
Oh, so like "applications *additionally* require"? If so, then i'd prefer that rather than the final phrase; but otherwise, i'm okay with leaving the latter in.

Reply:
I'm not sure I get you... Qt5 requires that stuff to use Wayland, but you can skip it and it just won't use Wayland. EFL requires that variable to work at all (or at least for cases when Xwayland is enabled, which will be most of them). I'm not sure how else I can phrase it.

Reply:
i'm sorry i'm not being clear. i'm just saying that if we say that setting an env var is *required*, then it's already implied that *not* setting it will cause issues, so the ", and can have issues" part of the sentence is not needed. But although i feel it's unnecessary, it doesn't hurt that much to leave it in, so i'm happy to leave it as-is.